doc/devguide.md

```diff
@@ -83,6 +83,9 @@ If you would like to develop an application using the Azure IoT SDK for C# (usin
 Contains the IoT Hub Device SDK source, unit-tests and samples. 
 This produces the `Microsoft.Azure.Devices.Client` NuGet package.
 
+```diff
+- .NET MicroFramework will not be supported in future versions of the SDK.
+```
 The src.NetMF folder contains the .NET Microframework port.
```

Comment:
Is this true for provisioning as well ?

Reply:
No: DPS was a new feature and so it is supported only for ns20. We will need to redo our top-level readme.md as we don't make support very well understood (table with actual support would be great) - not per feature but per DLL.